packages/sites/src/domains/add-domain.ts

```diff
@@ -20,7 +20,7 @@ export function addDomain(
   }
   const headers = _getAuthHeader(hubRequestOptions);
   headers["Content-Type"] = "application/json";
-  const url = `${_getDomainServiceUrl(hubRequestOptions)}?force=true`; // Use the force...
+  const url = `${_getDomainServiceUrl(hubRequestOptions)}`;
```

Comment:
Just curious. Why no `force=true`?

Reply:
this flag never worked in the original domain service, so we didn't implement it in the new one